### PR TITLE
Use VAS.Core.Catalog for translations

### PR DESCRIPTION
**Pull request by ylatuya**

### CODE REVIEW

File: LongoMatch.GUI/Gui/GUIToolkit.cs [hunk 1]
```diff
@@ -72,7 +72,7 @@ private GUIToolkit ()
 			}
 		}
 
-		public override void ExportFrameSeries (Project openedProject, TimelineEvent play, string snapshotsDir)
```
Comment (vguzmanp) [inline LongoMatch.GUI/Gui/GUIToolkit.cs]: This is not related to the catalog, right?

File: LongoMatch.Services/State/LMUpgradeLimitationState.cs [hunk 2]
```diff
@@ -23,17 +22,17 @@ protected override void CreateViewModel (dynamic data)
 			LimitationVM limitation = (LimitationVM)data.limitationVM;
 			if (data.limitationVM is FeatureLimitationVM) {
 				FeatureLimitationVM featureLimitation = (FeatureLimitationVM)limitation;
-				string featureMessage = (string.IsNullOrEmpty (featureLimitation.DetailInfo)) ? featureLimitation.DisplayName : featureLimitation.DetailInfo;
+				string featureMessage = (string.IsNullOrEmpty (featureLimitation.DetailInfo)) ?
+					featureLimitation.DisplayName : featureLimitation.DetailInfo;
 				if (!string.IsNullOrEmpty (featureLimitation.DetailInfo)) {
-					ViewModel.Header = $"{Catalog.GetString (featureLimitation.DetailInfo)}";
+					ViewModel.Header = featureLimitation.DetailInfo;
```
Comment (vguzmanp) [inline LongoMatch.Services/State/LMUpgradeLimitationState.cs]: Why do you remove the catalog from here?

File: LongoMatch.Services/State/LMUpgradeLimitationState.cs [hunk 3]
```diff
@@ -23,17 +22,17 @@ protected override void CreateViewModel (dynamic data)
...
-						$"{Catalog.GetString ("is not available in the")} {App.Current.LicenseManager.LicenseStatus.PlanName} " +
-						$"{Catalog.GetString ("plan")}";
+					ViewModel.Header = Catalog.GetString (
+						$"The {featureLimitation.DisplayName} is not available in the {App.Current.LicenseManager.LicenseStatus.PlanName} plan");
 				}
 			} else if (limitation is CountLimitationVM) {
-				ViewModel.Header = Catalog.GetString ("You have reached the limit of") +
-					$" {limitation.DisplayName} available for your plan";
+				ViewModel.Header = Catalog.GetString (
+					$"You have reached the limit of {limitation.DisplayName} available for your plan");
```
Comment (vguzmanp) [inline LongoMatch.Services/State/LMUpgradeLimitationState.cs]: Will translators understand the `{}` syntax? Or is it replaced when creating the potfiles?

Comment (ylatuya) [reply LongoMatch.Services/State/LMUpgradeLimitationState.cs]: It understands it, similar to when using indexed parameters {0} or {1}

File: LongoMatch.Services/State/LMUpgradeLimitationState.cs (hunk 2 above)
Comment (ylatuya) [reply LongoMatch.Services/State/LMUpgradeLimitationState.cs]: Catalog.GetString will only generate translatable string when using string literals, otherwise it's impossible to know what the value of Detailnfo.
This strings should be (and currently is) translated when setting it DetailInfo = Catalog.GetString("something")

File: LongoMatch.GUI/Gui/GUIToolkit.cs (hunk 1 above)
Comment (ylatuya) [reply LongoMatch.GUI/Gui/GUIToolkit.cs]: pushed from the wrong branch

File: LongoMatch.Services/State/LMUpgradeLimitationState.cs (hunk 2 above)
Comment (vguzmanp) [reply LongoMatch.Services/State/LMUpgradeLimitationState.cs]: 👌
